sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDFSuite.scala

```diff
@@ -652,6 +652,16 @@ class HiveUDFSuite extends QueryTest with TestHiveSingleton with SQLTestUtils {
       }
     }
   }
+  test("SPARK-25768 Hive UDF supports literal struct type") {
```

Comment:
`SPARK-25768`? This should be `SPARK-28012`. Also, please add a new line before line 655.